### PR TITLE
Proofreading our new CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jfly**
Some nits in here related to English and consistency. Nothing too major.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,16 +8,16 @@ The only requirement is being kind and mindful of others, all communication
 shall be done with respect and without any kind of discrimination nor aggresive
 language/attitudes towards anyone.
 
-# Contributing Code.
+# Contributing code
```
Comment (jfly) [inline CONTRIBUTING.md:11]: Just trying to achieve consistency in this file with the headers

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ If you are adding a new command, create a new test suite (that is a `.bats` file
 
 See how other tests are implemented, and use them as starting point. Use our `envrc_load`
 defined at `test_helpers.bash` in order to load the direnv environment on your test.
-Also, be sure to read about [`bats-core`](https://github.com/bats-core/bats-core) usage, most importantly it's `run` and `$output`.
+Also, be sure to read about [`bats-core`](https://github.com/bats-core/bats-core) usage, most importantly its `run` and `$output`.
```
Comment (jfly) [inline CONTRIBUTING.md:44]: "it's" = "it is", *not* a possessive. English is weird and confusing.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -53,12 +53,12 @@ freely where these tools come from on your local environment.
 
 ## Running tests
 
-Running `make` will perform all integration tests, check formatting and check for common shell problems in code.
+Running `make` will perform all integration tests, check formatting, and check for common shell problems in code.
```
Comment (jfly) [inline CONTRIBUTING.md:56]: I don't know if we care about the Oxford comma or not, but I think most coders prefer it.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -67,10 +67,8 @@ env ASDF_DIRENV_DEBUG=true  bats -x test/use_asdf.bats -f 'use multiple versions
 
 ## Getting help
 
-Your pull-request needs not to be perfect. If you feel like you've implemented your core idea
+Your pull-request does not need to be perfect. If you feel like you've implemented your core idea
```
Comment (jfly) [inline CONTRIBUTING.md:70]: > needs not to be perfect

I imagine we're okay with a perfect PR, haha. It's just that we're also okay with PRs that are not perfect.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -26,9 +26,9 @@ in order to stop on first error. Commands shall not source each other. The ony
 way to share code is to source other bash files outside of `lib/commands`.
 
 
-We are not strictly POSIX complaint, not seek to have it as a requirement
-but our CI test should at least run on Linux and MacOS. When possible, prefer
-commands and options that work on most platforms.
+We are not strictly POSIX complaint, nor seek to have it as a requirement, but
+we do want things to work on both Linux and MacOS (our CI runs on both OSes).
+When possible, prefer commands and options that work on most platforms.
```
Comment (jfly) [inline CONTRIBUTING.md:31]: This was the part I was least sure about. I think there is a missing word or typo in the original paragraph, and it's making it hard for me to understand what it exactly it was trying to say.

File: CONTRIBUTING.md (hunk 2 above)
Comment (vic) [reply CONTRIBUTING.md:49]: me being almost deaf, learned English by reading mailing-lists about parrot-vm :D (lurking on what perl people said between blessings and weird symbols) you can bet it, this one hits me frequently haha.

Thanks for fixing.

File: CONTRIBUTING.md (hunk 5 above)
Comment (vic) [reply CONTRIBUTING.md:31]: you got it perfect. I was trying to say that we might be as cross-platform as possible, while we don't *need* to be strictly POSIX complaint.

**Comment by vic** [comment]
Thank you so much !

File: CONTRIBUTING.md (hunk 2 above)
Comment (jfly) [reply CONTRIBUTING.md:49]: Ha! No problem.

**Pull request merged**